SnowplowSwiftDemo/Cartfile.resolved

```diff
@@ -1,3 +1,3 @@
-git "file:///Users/mhadam/Snowplow/snowplow-objc-tracker" "5ba3cebcfabc8483a00cd64f54098ea4a79b9b8c"
+git "file:///Users/mhadam/Snowplow/snowplow-objc-tracker" "73cfae47c5efb2b8c57c801e0ebde5508607139e"
```

Comment:
Just to confirm, it looks personal due to the full path, should it stay as is?

Reply:
I'll take care of this, thanks for pointing it out